Project/Sources/Classes/Base32.4dm

```diff
@@ -54,7 +54,12 @@ Function encode
 		End if 
 		$bits:=$bits-5
 	End while 
-	$0:=Replace string:C233($obuff; "="; "")
+	$obuff:=Replace string:C233($obuff; "="; ""; *)
+	
```

Comment:
otp is base32 url encoding , not pure base32, there is no =

Reply:
ah, I see 🙄😃

Reply:
and maybe this is a useful bit of information which deserves to be in the comment of the class or the function (I may be old fashion that way 😉)

Reply:
Yes code is not clear and could produce this misunderstanding.
I just do this lib fast without interest to base32, just need it 
In README.md there is link to otp rfc and this base32

The best is add option to make url or not base32 encoding
With less effort rename class or function to Base32URL or encodeURL